examples/boid_flockers/app.py

```diff
@@ -15,11 +15,12 @@ def boid_draw(agent):
     "separation": 2,
 }
 
+model = BoidFlockers(100, 100, 100, 5, 10, 2)
```

Comment:
Adding this here should allow you to reduce the `model_params` dict above, right?

Reply:
You're right! 

Haven't thought about this. Currently it actually requires all parameters, but we should reduce that to only the parameters that can be changed. Should actually simplify the logic

Reply:
Unfortunately its not so easy. On a parameter change we have to re-initialize the model. But for that we need to know the parameter values of the model initialization. But they are not retained after initialization. Lets think about this a bit more

Reply:
Sounds like we need 
- https://github.com/projectmesa/mesa/issues/2268